dm: add some missing translation (#13852)
pingcap/docs: dm/task-configuration-file-full.md

## Configuration order

> From the sample configuration file, you can see that the configuration file is divided into a total of two parts: `global configuration` and `instance configuration`, where the `global configuration` is further divided into `basic configuration` and `feature configuration set`. The configuration order is as follows:

Reviewer: ```suggestion
From the sample configuration file, you can see that the configuration file contains two parts: `Global setting` and `Instance configuration`, where the `Global setting` contains `Basic configuration` and `Feature configuration se`. The configuration order is as follows:
```

> 

Reviewer: Please also update "Global setting" in Line 21 to `Global configuration`
